ssml-plugin/build.gradle

```diff
@@ -10,7 +10,7 @@ apply from: '../docu.gradle'
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     implementation project(':core')
-    implementation project(':ssml')
+    implementation project(':ssml-builder')
```

Comment:
What's the reason of this change?

Reply:
This is an old change which I forgot to push. This is related to the way how the modules are pushed to jcenter.